platform/windows/native_menu_windows.cpp

```diff
@@ -1095,7 +1095,7 @@ void NativeMenuWindows::set_item_icon(const RID &p_rid, int p_idx, const Ref<Tex
 				item_data->bmp = _make_bitmap(item_data->img);
 			} else {
 				item_data->img = Ref<Image>();
-				item_data->bmp = nullptr;
+				item_data->bmp = (HBITMAP)INVALID_HANDLE_VALUE;
```

Comment:
We are unsetting icon here, and `INVALID_HANDLE_VALUE` will be equivalent to, so not sure if it's correct (we want empty bitmap, not custom draw):

> HBMMENU_CALLBACK ((HBITMAP) -1)
> A bitmap that is drawn by the window that owns the menu. The application must process the [WM_MEASUREITEM](https://learn.microsoft.com/en-us/windows/desktop/Controls/wm-measureitem) and [WM_DRAWITEM](https://learn.microsoft.com/en-us/windows/desktop/Controls/wm-drawitem) messages.

Reply:
Update: Other menu image related functions seem to use `NULL` as empty bitmap. So it's most likely should be `NULL` here as well.